arm/cookbooks/win_workstation/test/recipes/default_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'inspec'
+
+describe package 'git' do
```

Comment:
I GET IT NOW.
I keep forgetting that Chef can run tests against it's "internal idea" of what will happen in the real world.

Roger using ChefSpec for sanity cheking, and test-kitchen for more "real" testing.  This makes sense to me.

That helps a ton. Thanks Peter.